test/asm/purge-deferred.asm

```diff
@@ -0,0 +1,10 @@
+DEF prefix EQUS "cool"
```

Comment:
Ideally, a comment would indicate somewhere *what* the test is checking for. (This becomes a life-saver when some tests break from unrelated changes, like syntax breaks or bugfixes.)